content/en/developers/marketplace/offering.md

```diff
@@ -0,0 +1,260 @@
+- A [Datadog App](#build-a-datadog-app)
+- A [SaaS license or subscription](#build-a-saas-license-or-professional-service-integration)
+- [Professional services](#build-a-bi-directional-data-integration)
+
+### Build an Agent-based integration
+
+#### OpenMetrics check
+
+An [OpenMetrics check][2] is suitable for gathering telemetry data from existing applications that expose metrics using the OpenMetrics standard.
+
+
+#### Python Check
+
+A [Python check][3] is suitable for monitoring services or products that do not expose metrics in a standard format. The Python check is used to collect telemetry data from various APIs or command line tools.
+
+
+#### DogStatsD
+
+[DogStatsD][4] is suitable for applications that already emit telemetry using the StatsD protocol. 
+
+Datadog adds additional Datadog-specific extensions to the StatsD protocol, including the following:
+
+- Histogram Metric Type
+- Service Checks
+- Events
+- Tagging
+
+
+
+For more information about Datadog Agent-based integrations, see [Introduction to Agent-based Integrations][5].
+
+### Build a REST API integration
+
+Use an [API integration][6] to enrich and submit data from your backend, or pull data directly out of Datadog. API integrations work well in building a connector between Datadog and another SaaS platform.
+ 
+Since API integrations do not use the Datadog Agent to collect data, you need to create an [informational tile-only listing](#build-a-saas-license-or-professional-service-integration) once your development work is complete.
+ 
+A Datadog API key is required to submit data to a Datadog API endpoint, and an application key is required to query data from Datadog or create resources on the Datadog site. Optionally, you can setup [OAuth for a data integration][7] in a Marketplace tile instead.
+
+### Build a Datadog App
+
+[Datadog Apps][8] are custom dashboard widgets that are developed in the [Datadog Developer Platform][9]. Once your Datadog App is ready to publish, you need to create an [informational tile-only listing](#build-a-saas-license-or-professional-service-integration) on the **Integrations** or **Marketplace** page.
+
+### Build a SaaS license or professional service integration
+
+If you are developing an offering that does not use the Datadog Agent (such as a Datadog App, SaaS license, professional service, or an API-based integration), you only need to create an informational tile listing. These tile-only listings require Datadog users to install and configure the integration outside of Datadog, and listings for SaaS licenses and professional services do not collect any data.
+
+The Datadog Development Toolkit offers a command option to create tile-only scaffolding: `ddev create -t tile "<Offering Name>"`. When using this command, you only receive the files related to your tile, instead of all the files used to build a full Agent-based data integration.
+
+### Build a bi-directional data integration
```

Comment:
It's odd that this is the same weight as the headers above because all integrations should be bi-directional. Can we somehow combine this so that it applies to Agent and API based integrations?